### PR TITLE
Remove Hyrax::BasicMetadata for AllinsonFlex

### PR DESCRIPTION
**Pull request by orangewolf**
# Summary
Addresses #199 by removing Hyrax::BasicMetadata and modifying AllinsonFlex to temporarily define attributes which cannot be adequately supported as required in Hyrax via dynamic schema.

# Screenshots / Video

# Expected Behavior

# Notes

If you need further information on how to fill this out, read the [playbook](https://playbook-staging.notch8.com/en/notch8/process/merge-requests)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -8,7 +8,7 @@ GIT
 
 GIT
   remote: https://github.com/samvera-labs/allinson_flex.git
-  revision: 2c930adc0e4c88a6c5a44c4f43e5b9a6f41666fe
+  revision: 96b7c48da97698c4d09c63f9b8b1fcbe1f763b49
```
Comment (ShanaLMoore) [inline Gemfile.lock:11]: Revert the last two commits
Run:
bundle update allinson_flex --conservative 

We should only see an update to the revision in gemfile.lock

**Review by ShanaLMoore — CHANGES_REQUESTED** [review]
Nice work. This is working after making a few adjustments...

Locally I reverted the last two commits, updated allinson flex conservatively, corrected a rubocop issue, and needed to remove [this](https://github.com/scientist-softserv/utk-hyku/blob/main/app/indexers/app_indexer.rb#L10) line in order to successfully save any work type. 

Leaving that line in app indexer caused the creation of a work to fail. It is basically looking for all the properties defined in hyrax basic metadata, so since we removed most of them it caused issues. 

ref: https://github.com/scientist-softserv/utk-hyku/pull/168/files

**Comment by laritakr** [comment]
> #### ⚠️ GitGuardian has uncovered 2 secrets following the scan of your pull request.
> Please consider investigating the findings and remediating the incidents. Failure to do so may lead to compromising the associated services or software components.
> 
> 🔎 Detected hardcoded secrets in your pull request
> 🛠 Guidelines to remediate hardcoded secrets
> 🦉 [GitGuardian](https://dashboard.gitguardian.com/auth/login/?utm_medium=checkruns&utm_source=github&utm_campaign=cr1) detects secrets in your source code to help developers and security teams secure the modern development process. You are seeing this because you or someone else with access to this repository has authorized GitGuardian to scan your pull request.Our GitHub checks need improvements? [Share your feedbacks](https://050127dayho.typeform.com/to/KmeAPTMk)!

**These secrets warnings are brought back in due to merging the main branch updates into my branch. Nothing I changed included any secrets.**

File: app/indexers/app_indexer.rb [hunk 2]
```diff
@@ -6,8 +6,8 @@ class AppIndexer < Hyrax::WorkIndexer
   # include Hyrax::IndexesBasicMetadata
 
   # Fetch remote labels for objects with controlled properties (i.e. :based_near)
-  # You can remove this if you don't want this behavior
-  include Hyrax::IndexesLinkedMetadata
+  # Utk does not include based_near and does not need deep indexing.
+  # include Hyrax::IndexesLinkedMetadata
```
Comment (ShanaLMoore) [inline app/indexers/app_indexer.rb:10]: 👍🏿

File: app/indexers/attachment_indexer.rb [hunk 3]
```diff
@@ -3,14 +3,6 @@
 # Generated via
 #  `rails generate hyrax:work Attachment`
 class AttachmentIndexer < AppIndexer
-  # This indexes the default metadata. You can remove it if you want to
```
Comment (ShanaLMoore) [inline app/indexers/attachment_indexer.rb:6]: 👍🏿

File: Gemfile.lock (hunk 1 above)
Comment (ShanaLMoore) [inline Gemfile.lock:11]: 👍🏿

**Review by ShanaLMoore — APPROVED** [review]

**Pull request merged**